dataeng/resources/load-events-to-vertica.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+if [ -z "$NUM_REDUCE_TASKS" ]; then
+    NUM_REDUCE_TASKS=$(( $NUM_TASK_CAPACITY*2 ))
```

Comment:
Hmm.   Actually, isn't this also going to be zero?   Maybe this isn't so good.   And maybe --n-reduce-tasks should also be removed below if this task is only performing loading and not doing any Sqoop or other processing.